assets/src/edit-story/output/utils/styles.js

```diff
@@ -48,6 +55,14 @@ function CustomStyles() {
                 overflow: visible;
               }
 
+              @media (max-aspect-ratio: ${gridLayerExpandUpperBound})  {
+                @media (min-aspect-ratio: ${gridLayerExpandLowerBound}) {
+                  amp-story-grid-layer.grid-layer {
+                    margin-top: calc(${fullBleedHeight} * 100% - ${safeZoneHeight}px);
```

Comment:
I don't quite follow the logic here.

Let's review: `amp-story-grid-layer` is 2:3 (`PAGE_RATIO`), and has a 9:16 "full-bleed" child with that overflows it (`FULLBLEED_RATIO`). 

So we know that setting `margin-top: 0` is too small, since we'd be truncating half of (<full_bleed_height> - <safe_area_height>). So what value should we set `margin-top`?

Reply:
Fixed with the new calculation method. Thanks for the explanation to the `pageHeight = 100vw / PAGE_RATIO` part!